src/test/datascience/interactiveWindow.vscode.common.test.ts

```diff
@@ -0,0 +1,493 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { assert } from 'chai';
+import * as sinon from 'sinon';
+import * as vscode from 'vscode';
+import { traceInfo, traceInfoIfCI } from '../../platform/logging';
+import { getDisplayPath } from '../../platform/common/platform/fs-paths';
+import { IDisposable } from '../../platform/common/types';
+import { InteractiveWindowProvider } from '../../interactive-window/interactiveWindowProvider';
+import { IKernelProvider } from '../../platform/../kernels/types';
+import {
+    captureScreenShot,
+    createEventHandler,
+    IExtensionTestApi,
+    initialize,
+    startJupyterServer,
+    waitForCondition
+} from '../common';
+import {
+    createStandaloneInteractiveWindow,
+    insertIntoInputEditor,
+    runNewPythonFile,
+    submitFromPythonFile,
+    submitFromPythonFileUsingCodeWatcher,
+    waitForInteractiveWindow,
+    waitForLastCellToComplete
+} from './helpers';
+import {
+    assertHasTextOutputInVSCode,
+    clickOKForRestartPrompt,
+    closeNotebooksAndCleanUpAfterTests,
+    defaultNotebookTestTimeout,
+    waitForExecutionCompletedSuccessfully,
+    waitForExecutionCompletedWithErrors,
+    waitForTextOutput
+} from './notebook/helper';
+import { translateCellErrorOutput, getTextOutputValue } from '../../notebooks/helpers';
+import { INotebookControllerManager } from '../../notebooks/types';
+import { IInteractiveWindowProvider } from '../../interactive-window/types';
+import { IInterpreterService } from '../../platform/interpreter/contracts';
+import { areInterpreterPathsSame } from '../../platform/pythonEnvironments/info/interpreter';
+import { IS_REMOTE_NATIVE_TEST } from '../constants';
+import { sleep } from '../core';
+import { IPYTHON_VERSION_CODE } from '../constants';
+
+suite(`Interactive window`, async function () {
```

Comment:
Interactive window tests for the web, very little work to enable this (once I enabled IW debugger tests for the web)